mgmt2/rpc/jsonrpc/Context.cc

```diff
@@ -0,0 +1,35 @@
+/**
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+#include "Context.h"
+
+namespace rpc
+{
+// --- Call Context impl
+ts::Errata
+Context::Auth::any_blockers(TSRPCHandlerOptions const &options) const
+{
+  ts::Errata out;
+  // check every registered callback and see if they have something to say. Then report back to the manager
+  for (auto &&check : _checkers) {
```

Comment:
Shouldn't `check` return a `bool` rather than taking a `bool&`? Does it return something else that's being ignored?